### ISSUE TITLE
请问是基于官方V22.6.6.57的版本修改的吗？

### ISSUE THREAD

**Issue by zhaibin18** [JimLee1996/K2P-FW#4]
在官方版本的web里可以升级吗？还是需要从breed升级？

**Comment by JimLee1996** [comment]
是的。因为去掉了image header，所以只能在breed里面升级了。如果出了新的固件，可以参考我的commits对新固件进行优化

**Comment by zhaibin18** [comment]
使用官方新版本22.6.6.57固件修改后，在breed中刷说固件无法判断的类型。

**Comment by zhaibin18** [comment]
![1](https://user-images.githubusercontent.com/6645145/32604851-23d532f0-c58a-11e7-8a9e-91005af3a56a.png)

**Comment by JimLee1996** [comment]
breed 里面刷机时，分区选择斐讯分区，而不是默认的公版
